llmfoundry/models/hf/model_wrapper.py

```diff
@@ -45,24 +47,19 @@ def __init__(self,
                  eval_metrics: Optional[List[Metric]] = None,
                  z_loss: float = 0.0,
                  shift_labels: bool = False,
-                 init_device: Optional[str] = None):
+                 init_device: Optional[str] = None,
+                 peft_config: Optional['PeftConfig'] = None):
         super().__init__(model,
                          tokenizer,
                          use_logits=True,
                          metrics=metrics,
                          eval_metrics=eval_metrics,
-                         shift_labels=shift_labels)
+                         shift_labels=shift_labels,
+                         peft_config=peft_config)
```

Comment:
will need value passed for `should_save_peft_only`

Reply:
My intention was not to support that configuration option via LLM Foundry, and just always save the PEFT weights only (defaults to True).

Reply:
gotcha, can you pass true explicitly then?

Reply:
sure